onnx_tf/handlers/frontend/size.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import tensorflow as tf
+
+from onnx import mapping
+from onnx_tf.common import get_unique_suffix
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_op
+
+
+@onnx_op("Size")
+@tf_op("Size")
+class Size(FrontendHandler):
+
+  @classmethod
+  def version_1(cls, node, **kwargs):
+    # tf.size out_type could be int32 or int64
+    need_cast = node.attr['out_type'] == tf.int32
+
+    size_suffix = "_" + get_unique_suffix() if need_cast else ""
+    size_output_name = cls.get_outputs_names(node)[0] + size_suffix
+    size_node = cls.make_node_from_tf_node(
+        node, [node.inputs[0]],
+        outputs=[size_output_name],
+        name=node.name + size_suffix)
+
+    if not need_cast:
+      return [size_node]
+
+    dst_t = mapping.NP_TYPE_TO_TENSOR_TYPE[np.dtype(
+        tf.as_dtype(tf.int32).as_numpy_dtype)]
```

Comment:
Will not work if user want to export an opset v1 model.

Reply:
The updated code is supposed to work with opset v1. But I found a logic flaw in determining the frontend handler.SINCE_VERSION. In this case, size is in ONNX opset v1 and never changed, while cast is in ONNX opset v1 and changed in v6. Now I need to use version_6 in my handler when opset >= v6 however get version_1 because of the logic in the handler_helper. We might need a common solution for the cases that a frontend handler invokes multiple ONNX operators which has different opset versions.

Reply:
My 2 cent: @fumihwh and I went over this issue a few days back. Some thoughts:
Firstly, @fumihwh started to call handlers of auxiliary nodes directly, c.f. how Transpose handler is used here: https://github.com/onnx/onnx-tensorflow/blob/93954860a66eab98f488da724da488a85e149166/onnx_tf/handlers/frontend/conv_mixin.py. Although I think this code can be improved as well, since here we are passing in a onnx node where the frontend handler expects a tf node. No big technical difference, but we in the future may find it less confusing looking back.

Secondly, we should modify how handlers behave to handle the complication you described. In my opinion, we should add another parameter to the `handle` method of handler called "up_to_version" which specify maximum op version handler is allow to produce. Then to create an auxiliary node, we should call that handler's `handle` function and pass in either 

1. the opset version user provide through API or CLI, 
2. or the SINCE_VERSIOIN that the op handler corresponds to. 

The latter seems simpler to implement, but the former will give user a maximally up-to-date model without breaking user specified opset constraint.

Reply:
@tjingrant @fumihwh  Thanks for the comments. I changed the code to call cast handler directly. It works, just a little strange to construct a ONNX node to feed the other frontend handler.